src/Core/src/Platform/Gtk/MauiToolbar.cs

```diff
@@ -0,0 +1,25 @@
+using Gtk;
+
+namespace Microsoft.Maui.Platform
```

Comment:
Shouldn't the namespace be Microsoft.Maui.Platform.GTK?

Reply:
Nope, I don't think so. There was already a Tizen implementation and that has Microsoft.Maui.Platform as its namespace too.

Reply:
Resolve?